torch/csrc/jit/codegen/cuda/shape_inference.cpp

```diff
@@ -386,47 +367,37 @@ class NaiveTypePropagator {
       }
       case aten::sum_to_size:
       case aten::_grad_sum_to_size: {
-        auto out_type = node->input(0)->type()->cast<TensorType>();
+        auto out_type = getTensorType(node, 0);
```

Comment:
note to myself, this is not your problem, I'm not sure how we are not specifying output dimensions here, we clearly should. Can you add a TODO here?

Reply:
I used the size information from argument 1 here.